sat/cli/status/status_module.py

```diff
@@ -553,12 +552,22 @@ def rows(self):
                     try:
                         component_session = bos_client.get_session(session_id)
                     except APIError as err:
-                        LOGGER.warning('Could not retrieve BOS session for component %s: %s',
-                                       component_xname, err)
+                        LOGGER.debug('Could not retrieve BOS session for component %s: %s',
+                                     component_xname, err)
                     self._cached_bos_sessions[session_id] = component_session
                 if component_session is None:
                     continue
 
+                # Track missing sessions
```

Comment:
This block of code is tracking the opposite of what we want to track. That is, it is tracking the sessions which do exist in BOS and then reporting on those as if they're missing. For example:

```
ncn-m001:~/haasken # sat --loglevel debug status --bos-fields
...
DEBUG: Issuing GET request to URL 'https://api-gw-service-nmn.local/apis/bos/v2/sessions/99194948-b639-489b-8d9b-a41f4fce4187'
DEBUG: Received response to GET request to URL 'https://api-gw-service-nmn.local/apis/bos/v2/sessions/99194948-b639-489b-8d9b-a41f4fce4187' with status code: '200': OK
DEBUG: Session id 99194948-b639-489b-8d9b-a41f4fce4187 which applies to 1 nodes is missing
...
+----------------+-------------+--------------------------------------+------------------------------------------------------------+--------------------------------------+
| xname          | Boot Status | Most Recent BOS Session              | Most Recent Session Template                               | Most Recent Image                    |
+----------------+-------------+--------------------------------------+------------------------------------------------------------+--------------------------------------+
| x8000c0s0b0n0  | stable      | 99194948-b639-489b-8d9b-a41f4fce4187 | ssi-compute-23.11.0-beta.6.x86_64-csm-1.5.0.beta76-iw5a-1  | bbb12a16-5e80-4733-9377-05009a136d51 |
...
```

It should instead be placed in the `if component_session is None:` block above.